data/remnant ships.txt

```diff
@@ -413,6 +493,62 @@ ship "Pelican"
 
 
 
+ship "Petrel"
+	sprite "ship/petrel"
+	thumbnail "thumbnail/petrel"
+	attributes
+		category "Fighter"
+		licenses
+			Remnant
+		"cost" 4500000
+		"shields" 2000
+		"hull" 2500
+		"cargo space" 20
+		"fuel capacity" 100
+		"ramscoop" 1
+		"required crew" 1
+		"bunks" 3
+		"mass" 50
+		"drag" 1.2
+		"heat dissipation" .9
+		"outfit space" 195
+		"shield generation" 1.9
+		"shield energy" 1.1
+		"hull repair rate" 0.9
+		"hull energy" 0.6
+		"outfit scan power" 12
+		"outfit scan speed" 1
+		"tactical scan power" 26
+		"asteroid scan power" 30
+		"weapon capacity" 80
+		"engine capacity" 55
```

Comment:
Weapon capacity and engine capacity positions out of order.

Reply:
Fixed.

Reply:
No, the order was fine, but these should be up with outfit space.

Reply:
OK, moved them up with the outfit space.